### PR TITLE
[audiodsp] Use label/label2/icon instead of properties

### PR DESCRIPTION
**Pull request by BigNoid**
To be consistent I changed the list properties to common label/label2/icon.
Also added a check if the icon is empty to use `DefaultAddonAudioDSP.png` instead so no fallbacks have to be used in skin code.

@phil65 @AchimTuran

### CODE REVIEW

File: xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp [hunk 1]
```diff
@@ -1026,14 +1026,17 @@ CFileItem *CGUIDialogAudioDSPManager::helper_CreateModeListItem(CActiveAEDSPMode
   // set list item properties
   pItem->SetProperty("ActiveMode", isActive);
   pItem->SetProperty("Number", number);
-  pItem->SetProperty("Name", modeName);
+  pItem->SetLabel2(modeName);
```
Comment (AlwinEsch) [inline xbmc/settings/dialogs/GUIDialogAudioDSPManager.cpp]: Looks good, but maybe swap Label2 with Label, the Mode name is more the primary in DSP Manager.

**Comment by BigNoid** [comment]
@AlwinEsch Ah okay. I just followed what Estuary used as primary as I am not that into audio dsp.
Updated.

**Comment by phil65** [comment]
Makes sense. Thx!

**Comment by AchimTuran** [comment]
Look good thanks.

**Comment by BigNoid** [comment]
jenkins build this please

**Pull request merged**